services/core-api/tests/factories.py

```diff
@@ -43,6 +44,19 @@
 FACTORY_LIST = []
 
 
+def create_mine_and_permit(mine_kwargs={},
```

Comment:
Had a discussion with @Jsyro about this approach for factory generation. We could rejig this to create a permit through the permit amendment relationship in a way that's more consistent with the rest of our factory creators, but this approach does the trick. 

Just don't want this to be the example that drives other factory implementations if it's not necessary. Maybe a comment to that effect?